cmd/exercise/exercise.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/Contrast-Security-OSS/go-test-bench/pkg/servegin/gintest"
+	"github.com/Contrast-Security-OSS/go-test-bench/pkg/servestd/stdtest"
+)
+
+func exercise(addr string) error {
+	client := http.DefaultClient
+
+	// Send request to app root to determine framework
+	res, err := client.Get("http://" + addr)
```

Comment:
not suggesting to change the current implementation, just a side note: may be we want to add HTTPS test server